sdk/python/pyproject.toml

```diff
@@ -1,5 +1,6 @@
 [project]
 name = "freewayai"
+version = "0.1.4"
```

Comment:
You should remove this now. Versions get tagged with the Github release and not in the pyproject file. Once you remove it though this looks good to merge in.